run_scripts/2_fetch_raw_data.py

```diff
@@ -70,43 +69,78 @@
     storage_client = storage.Client.from_service_account_json(google_cloud_credentials_file_path)
     credentials_bucket = storage_client.bucket(bucket_name)
     credentials_file = credentials_bucket.blob(blob_name)
-    rapid_pro_token = credentials_file.download_as_string().strip()
+    rapid_pro_token = credentials_file.download_as_string().strip().decode("utf-8")
     print("Downloaded Rapid Pro token.")
 
+    with open(uuid_table_path) as f:
+        phone_number_uuid_table = PhoneNumberUuidTable.load(f)
+
+    with open(TEST_CONTACTS_PATH) as f:
+        test_contacts = json.load(f)
```

Comment:
Log the number of test contacts loaded

Reply:
no longer necessary because these are now read directly from configuration